app/code/Magento/InventoryInStorePickupAdminUi/Controller/Adminhtml/Order/NotifyPickup.php

```diff
@@ -77,26 +77,22 @@ public function execute(): ResultInterface
     {
         $order = $this->initOrder();
 
-        if ($order) {
-            try {
-                $this->notifyOrderIsReadyForPickup->execute((int)$order->getEntityId());
-                $this->messageManager->addSuccessMessage(__('The customer have been notified and shipment created.'));
-            } catch (LocalizedException $e) {
-                $this->messageManager->addErrorMessage($e->getMessage());
-            } catch (Exception $e) {
-                $this->messageManager->addErrorMessage(__('We can\'t notify the customer right now.'));
-                $this->logger->critical($e);
-            }
-
-            return $this->resultRedirectFactory->create()->setPath(
-                'sales/order/view',
-                [
-                    'order_id' => $order->getEntityId()
-                ]
-            );
+        try {
+            $this->notifyOrderIsReadyForPickup->execute((int)$order->getEntityId());
+            $this->messageManager->addSuccessMessage(__('The customer have been notified and shipment created.'));
+        } catch (LocalizedException $e) {
+            $this->messageManager->addErrorMessage($e->getMessage());
+        } catch (Exception $e) {
+            $this->messageManager->addErrorMessage(__('We can\'t notify the customer right now.'));
+            $this->logger->critical($e);
         }
 
-        return $this->resultRedirectFactory->create()->setPath('sales/*/');
```

Comment:
Seems Okay, but what is the point for changing these lines?

Reply:
this never will be called, since order always will be present